src/qml/LocatorItem.qml

```diff
@@ -10,6 +10,11 @@ Item {
   property bool searching: false
   height: childrenRect.height
 
+  function leaveSearching() {
```

Comment:
Can you add a docstring when this should be called?

Reply:
done

Reply:
Just wondering.
Should we use

```
onSearchingChanged {
    if ( !searching )
        focus = false
}
```
 instead? That would effectively avoid misusage

Reply:
done